src/locales/ja/common.json

```diff
@@ -1213,7 +1213,7 @@
     "transfer": "Transfer",
     "manager": "Manager",
     "settings": "Settings",
-    "platform": "Browse",
+    "platform": "Discover",
```

Comment:
changes not to be done on the locales* other than en, otherwise crowdin would consider it as "translated".

thanks.

Reply:
(these file are auto generated and crowdin will send automatically a PR to make them changed to "discover" too)